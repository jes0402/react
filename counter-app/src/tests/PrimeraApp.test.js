import React from 'react';
import { shallow } from 'enzyme'
import PrimeraApp from '../PrimeraApp'
import '@testing-library/jest-dom';


describe ("pruebas en <PrimeraApp/>", () => {
    test ("debe de mostrar <PrimeraApp/> correctamente", () => {
      
        const saludo = "Hola soy coco";

        const wrapper = shallow(<PrimeraApp saludo={ saludo} />);

        expect(wrapper).toMatchSnapshot();

    })
 });
 test ("debe de mostrar el subtitulo enviado por props", () => {
      
    const saludo = "Hola soy coco";
    const subTitulo = "Hola soy subtitulo";

    const wrapper = shallow(
    <PrimeraApp
         saludo={ saludo } 
         subtitulo={ subTitulo } 
    />
    );
    const textoParrafo = wrapper.find("p").text()
     expect(textoParrafo).toBe(subTitulo);

})