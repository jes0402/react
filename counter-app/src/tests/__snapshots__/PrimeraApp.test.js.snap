// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <PrimeraApp/> debe de mostrar <PrimeraApp/> correctamente 1`] = `
<Fragment>
  <h1>
     
    Hola soy coco
     
  </h1>
  <h1>
    Hola mundo
  </h1>
  <p>
    soy un subtitulo
  </p>
</Fragment>
`;
